PopPUNK/refine.py

```diff
@@ -90,6 +99,15 @@ def refineFit(distMat, sample_names, start_s, mean0, mean1,
     sys.stderr.write("Decision boundary starts at (" + "{:.2f}".format(start_point[0])
                       + "," + "{:.2f}".format(start_point[1]) + ")\n")
 
+    # load CUDA libraries
+    if use_gpu and not gpu_lib:
+      sys.stderr.write('Unable to load GPU libraries; exiting\n')
+      sys.exit(1)
+
+    # Set memory management for large networks
+    if use_gpu:
+      cudf.set_allocator("managed")
+
```

Comment:
This is repeated from the other file so should probably be a function

Reply:
Done in 2065a89.